chains/solana/contracts/programs/ccip-router/src/instructions/v1/onramp.rs

```diff
@@ -370,7 +371,11 @@ fn hash(msg: &SVM2AnyRampMessage) -> [u8; 32] {
         &header_sequence_number,
         &header_nonce,
         &msg.fee_token.to_bytes(),
-        &msg.fee_token_amount.to_be_bytes(),
+        // The cross-chain amounts are encoded in little endian, but
+        // this is irrelevant to the hashing function as long as both
+        // sides agree.
+        &msg.fee_token_amount.to_bytes(),
+        &msg.fee_value_juels.to_bytes(),
```

Comment:
agreed, we can use either encoding here (it's transparent to users and only affects our offchain code), whichever you think makes it easier for the offchain side or is most consistent with how we encode other fields is fine by me :+1:

Reply:
Per https://github.com/smartcontractkit/chainlink-ccip/pull/460#discussion_r1922517197, it's completely fine by me if you want to change this to BE here

Reply:
After your explanation [here](https://github.com/smartcontractkit/chainlink-ccip/pull/460#discussion_r1922559601) I'd rather stay in LE to be internally consistent: Better to stay close to the solana/borsh assumptions I think.